cookie_consent/middleware.py

```diff
@@ -26,7 +31,13 @@ def process_response(self, request, response):
                 continue
             group_version = cookie_dic.get(cookie_group.varname, None)
             for cookie in cookie_group.cookie_set.all():
-                if cookie.name not in request.COOKIES:
+                if cookie.name not in str(request.COOKIES):
+                    continue
+                if group_version == None and not settings.COOKIE_CONSENT_OPT_OUT:
+                    response.delete_cookie(
+                        smart_str(cookie.name),
```

Comment:
I don't understand why `smart_str` is needed, as the `name` field is a string already since it's a `models.CharField`

Reply:
Well I think making any change here depends on if the NAME = b"cookie_consent" setting from the conf.py file removes the bytes completely. Otherwise, it smart_str will probably need to stay.

Reply:
this is not the name of the cookie set in the browser with the accepted/rejected cookies, but the name of a cookie, such as `_ga` for google analytics. It appears on the right hand side of the cookie header, in this pattern: `{settings.COOKIE_CONSENT_NAME}={cookie.name}=-1`.

Because it is a models.CharField, the value is always `str` and not `bytes` and there is no need for `smart_str`.

Reply:
Ok then let's remove it. Let me know if you need me to change it. But I think since you have temporary library maintainer rights, you should be able to make edits, right? If you do, feel free to change it.